extensions/pipedream/README.md

```diff
@@ -0,0 +1,39 @@
+# Pipedream Workflow Manager for Raycast
+
+Manage your Pipedream workflows directly from Raycast. Add, monitor, and control your workflows with ease, including error tracking and quick actions.
+
+## Features
+
+- Add new Pipedream workflows to manage
+- View and manage saved workflows
+- Monitor workflow status and errors from the menu bar
+- Quick actions to activate/deactivate workflows
+- Error tracking and easy access to error logs
+- Sort and filter workflows based on various criteria
+
+## Setup
+
+1. Install the extension from the Raycast store.
+2. You'll need a Pipedream API key to use this extension. To get one:
+   - Log in to your Pipedream account
+   - Go to [Pipedream Account Settings](https://pipedream.com/settings/account)
```

Comment:
This link doesn't go to the Account section. Either fix it or tell the user in text.